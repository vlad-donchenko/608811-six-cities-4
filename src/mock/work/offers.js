const offers = [
  {
    title: `Beautiful & luxurious apartment at great location`,
    picture: `img/apartment-01.jpg`,
    price: 120,
    type: `Apartment`,
    rating: 4,
    isPremium: true,
    isBookmark: true
  },
  {
    title: `Wood and stone place`,
    picture: `img/apartment-01.jpg`,
    price: 220,
    type: `Apartment`,
    rating: 3,
    isPremium: false,
    isBookmark: false
  },
  {
    title: `Canal View Prinsengracht`,
    picture: `img/apartment-01.jpg`,
    price: 150,
    type: `Apartment`,
    rating: 2,
    isPremium: false,
    isBookmark: false
  },
  {
    title: `Nice, cozy, warm big bed apartment`,
    picture: `img/apartment-01.jpg`,
    price: 850,
    type: `Apartment`,
    rating: 5,
    isPremium: false,
    isBookmark: true
  },
  {
    title: `Wood and stone place`,
    picture: `img/apartment-01.jpg`,
    price: 450,
    type: `Apartment`,
    rating: 4.5,
    isPremium: true,
    isBookmark: false
  }
];

export {offers};
