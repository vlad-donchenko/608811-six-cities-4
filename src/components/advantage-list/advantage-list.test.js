import React from "react";
import renderer from 'react-test-renderer';
import Advantage from "./advantage-list.jsx";
import {offers} from "../../mock/test/offers";

it(`Should render Advantage`, () => {
  const tree = renderer.create(<Advantage list={offers[0].advantages}/>).toJSON();

  expect(tree).toMatchSnapshot();
});
