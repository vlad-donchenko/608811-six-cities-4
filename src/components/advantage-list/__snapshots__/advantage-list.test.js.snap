// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render Advantage 1`] = `
<ul
  className="property__inside-list"
>
  <li
    className="property__inside-item"
  >
    Coffee machine
  </li>
  <li
    className="property__inside-item"
  >
    Dishwasher
  </li>
  <li
    className="property__inside-item"
  >
    Towels
  </li>
  <li
    className="property__inside-item"
  >
    Baby seat
  </li>
  <li
    className="property__inside-item"
  >
    Cabel TV
  </li>
</ul>
`;
